api/src/main/java/org/openmrs/User.java

```diff
@@ -593,5 +671,127 @@ public Integer getId() {
 	public void setId(Integer id) {
 		setUserId(id);
 	}
+
+	@Override
+	public User getCreator() {
+		return creator;
+	}
+
+	@Override
+	public void setCreator(User creator) {
+		this.creator = creator;
+<<<<<<< HEAD
+		
+=======
+>>>>>>> 7f1edc92d80496f5d1c86e9e0b968292c5ebf238
```

Comment:
These issues all arise because I think, of the version I committed here. Here and below, we should go with option 2.

```suggestion
```

Reply:
Option 2 is, indentally, leaving that space blank.